Add minimal API sample of reading the request stream (#24327)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/minimal-apis.md

  }
  ```

> ### Read a request body

Author: ```suggestion
### Read the request body
```

> ### Reading request body

Reviewer: Localizes better according to our loc guidance folks ...

```suggestion
### Read a request body
```

> The request body can be read directly using a `HttpContext` or `HttpRequest` parameter.

Reviewer: Imperative tone (a direct instruction) avoids the state verb (be/being) ...

```suggestion
Read a request body directly using a <xref:Microsoft.AspNetCore.Http.HttpContext> or <xref:Microsoft.AspNetCore.Http.HttpRequest> parameter:
```

> Read a request body directly using a <xref:Microsoft.AspNetCore.Http.HttpContext> or <xref:Microsoft.AspNetCore.Http.HttpRequest> parameter:

Author: ```suggestion
Read the request body directly using a <xref:Microsoft.AspNetCore.Http.HttpContext> or <xref:Microsoft.AspNetCore.Http.HttpRequest> parameter:
```

[!code-csharp[](minimal-apis/samples/WebMinAPIs/Program.cs?name=snippet_fileupload)]

The preceding code:

> * Accesses the request body using `HttpRequest.Body`.

Reviewer: ```suggestion
* Accesses the request body using <xref:Microsoft.AspNetCore.Http.HttpRequest.Body?displayProperty=nameWithType>.
```

> * Accesses the request body using <xref:Microsoft.AspNetCore.Http.HttpRequest.Body?displayProperty=nameWithType>.

Author: ```suggestion
* Accesses the request body using <xref:Microsoft.AspNetCore.Http.HttpRequest.BodyReader?displayProperty=nameWithType>.
```
